worlds/EX/6/3.wbt

```diff
@@ -0,0 +1,61 @@
+#VRML_SIM R2023b utf8
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```

Comment:
Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")